lib/loggly/client.js

```diff
@@ -23,6 +23,17 @@ function stringify(msg) {
   return payload;
 }
 //
+// function to truncate message over 1 MB
+// 
+function truncateLargeMessage(message) {  
+  var MaximumBytesAllowedToLoggly = 1000 * 1000;
```

Comment:
Ideally good not to hard code the MaximumBytesAllowedToLoggly. It will be good to have the size variable and can be passed as config. For now I am approving this. @Shwetajain148  please log JIRA to make the value define in config.